src/generic/upgrade-notification/UpgradeNotification.jsx

```diff
@@ -309,6 +311,12 @@ const UpgradeNotification = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
+    if (currentSidebar === 'NOTIFICATIONS' && upgradeBtnRef.current) {
```

Comment:
[nit] You can use constant https://github.com/raccoongang/frontend-app-learning/blob/257c9dcd7f853d371d382e8f41fdebc1c24b743f/src/constants.js#L50

Reply:
Unfortunately, we don't have this constant in this learning branch ([open-release/palm.4](https://github.com/raccoongang/frontend-app-learning/tree/open-release/palm.4)). I agree that it is quite reasonable to use this value from the constant and I would like to replace this value in all files. But since we use overrides, I would prefer not to do this.

What do you think?

Reply:
Overrides it's just a method how to override templates, but for being more close to code it's good to use variables.

You can reuse this variable from here: https://github.com/raccoongang/frontend-app-learning/blob/open-release/palm.4/src/courseware/course/sidebar/sidebars/discussions/index.js#L2C30-L2C32

Reply:
decided to leave it as is